sysext.just

```diff
@@ -16,6 +16,9 @@ enable_repos := ""
 # Do not enable any Copr repos by default
 copr_repos := ""
 
+# Forcefully install from enabled repos instead of Fedora repos
+force_repos := ""
```

Comment:
We have an `enable_repos` knob above. Should we make this a `disable_repos` one instead?